### ISSUE TITLE
`Mono: Cannot create log file`

### ISSUE THREAD

**Issue by ScepticDope** [godotengine/godot#28388]
**Godot version:**
3.1.1 RC1

**OS/device including version:**
Windows 8.1

**Issue description:**
I get this error message when running just any project:
```
E 0:00:00:0309   Mono: Cannot create log file
  <C Source>     modules/mono/mono_gd/gd_mono_log.cpp:183 @ initialize()
```

Everything works fine without this log, but I thought I should report it. This happens with both the MSBuild Mono build tools and the MSBuild VS Build Tools.

**Steps to reproduce:**
Open a project and play the project.

**Comment by ScepticDope** [comment]
Just noticed it also gives the error in the debug window when starting the Godot Editor itself.

**Comment by nazgulsenpai** [comment]
Same issue here with 3.1.1 RC. Also had the same behavior in Master compiled 4 days ago. Rebuilding now to see if it persists. Started after #28139

Windows 10 Pro x64 1803
Mono x64 5.20.1.19 

```
OpenGL ES 3.0 Renderer: Radeon(TM) RX 460 Graphics
ERROR: initialize: Mono: Cannot create log file
   At: modules/mono/mono_gd/gd_mono_log.cpp:183
Mono: No log file, using default log handler
Mono: Config attempting to parse: 'D:\GoogleDrive\Godot\Godot_v3.1.1-rc1_mono_win64/GodotSharp/Mono/etc\mono\config'.
Mono: Image addref mscorlib[000000001B85E980] (asmctx DEFAULT) -> D:\GoogleDrive\Godot\Godot_v3.1.1-rc1_mono_win64/GodotSharp/Mono/lib/mono/4.5/mscorlib.dll[000000001B9CABA0]: 2
Mono: Prepared to set up assembly 'mscorlib' (D:\GoogleDrive\Godot\Godot_v3.1.1-rc1_mono_win64/GodotSharp/Mono/lib/mono/4.5/mscorlib.dll)
Mono: AOT: image 'D:\GoogleDrive\Godot\Godot_v3.1.1-rc1_mono_win64/GodotSharp/Mono/lib/mono/4.5/mscorlib.dll.dll' not found: The system cannot find the file specified.

Mono: AOT: image 'D:\GoogleDrive\Godot\Godot_v3.1.1-rc1_mono_win64/GodotSharp/Mono/lib/mono/aot-cache/amd64/mscorlib.dll.dll' not found: The system cannot find the path specified.

Mono: Assembly mscorlib[000000001B85E980] added to domain GodotEngine.RootDomain, ref_count=1
Mono: Assembly mscorlib[000000001B85E980] added to domain GodotEngine.ToolsDomain, ref_count=2
Mono: Image addref GodotSharpTools[000000001B85F0D0] (asmctx DEFAULT) -> D:\GoogleDrive\Godot\Godot_v3.1.1-rc1_mono_win64/GodotSharp/Tools/GodotSharpTools.dll[000000001BA70870]: 2
Mono: Prepared to set up assembly 'GodotSharpTools' (D:\GoogleDrive\Godot\Godot_v3.1.1-rc1_mono_win64/GodotSharp/Tools/GodotSharpTools.dll)
Mono: Assembly GodotSharpTools[000000001B85F0D0] added to domain GodotEngine.ToolsDomain, ref_count=1
Mono: Assembly mscorlib[000000001B85E980] added to domain GodotEngine.ScriptsDomain, ref_count=3

```

**Comment by nazgulsenpai** [comment]
Just an update, fresh build from today (e1d16e7) still has this issue.

**Comment by HysanDev** [comment]
Same issue here with 3.1.1 RC. Cannot create log file

**Comment by neikeq** [comment]
It will be fixed in the next 3.1.1 release.

**Comment by PabloMonfort** [comment]
hi there im just new at godot come from unity, and i have this error after open the first demo for test the navmesh example , and i try to build after install it for test and it faild and i cant understand whats the problem since i didnt even touch anything so i come here to tell you people about my bug,

here is my full log file
///////
onfig attempting to parse: 'D:\Downloads\Godot_v3.1.1-stable_mono_win64/GodotSharp/Mono/etc\mono\config'. (in domain Mono, info)
Image addref mscorlib[0000000021D6F490] (asmctx DEFAULT) -> D:\Downloads\Godot_v3.1.1-stable_mono_win64/GodotSharp/Mono/lib/mono/4.5/mscorlib.dll[0000000021EE3680]: 2 (in domain Mono, info)
Prepared to set up assembly 'mscorlib' (D:\Downloads\Godot_v3.1.1-stable_mono_win64/GodotSharp/Mono/lib/mono/4.5/mscorlib.dll) (in domain Mono, info)
AOT: image 'D:\Downloads\Godot_v3.1.1-stable_mono_win64/GodotSharp/Mono/lib/mono/4.5/mscorlib.dll.dll' not found: El sistema no puede encontrar el archivo especificado.
 (in domain Mono, info)
AOT: image 'D:\Downloads\Godot_v3.1.1-stable_mono_win64/GodotSharp/Mono/lib/mono/aot-cache/amd64/mscorlib.dll.dll' not found: El sistema no puede encontrar la ruta especificada.
 (in domain Mono, info)
Assembly mscorlib[0000000021D6F490] added to domain GodotEngine.RootDomain, ref_count=1 (in domain Mono, info)
Assembly mscorlib[0000000021D6F490] added to domain GodotEngine.ToolsDomain, ref_count=2 (in domain Mono, info)
